README.md

```diff
@@ -1 +1,28 @@
-# trailmap
\ No newline at end of file
+# trailmap
+
+This tool takes a Cyclus input file and conducts acquisition pathway analysis on the facilities specified.
+
+## Dependencies
+
+* Python 3
+* [Cyclus](https://github.com/cyclus/cyclus)
+* [Cycamore](https://github.com/cyclus/cycamore)
+* [NetworkX](https://networkx.github.io/)
+* [PyGraphviz](https://pygraphviz.github.io/)
+* [pydot](https://github.com/pydot/pydot)
```

Comment:
do you still need all of those ?
I am not sure Cycamore is required if you have other archetypes (it seems required for testing tho...)

I only see import from `json` `sys` `os` and `cyclus`
and `subprocess` and `xml` added in #15

Reply:
They will be used in the files I'm writing now. Should I wait until that PR to add them into the README?